napari_aicssegmentation/view/workflow_select_view.py

```diff
@@ -1,63 +1,217 @@
+        self.combo_channels = channels_dropdown.widget
+        layer_channel_selections = self._form_layout([layers_dropdown, channels_dropdown])
+
+        # Add all widgets
+        widgets = [
+            workflow_selection_title,
+            self.load_image_warning,
+            layer_channel_selections,
+        ]
+        for widget in widgets:
+            layout.addWidget(widget)
+        self._add_step_3_layout(layout, enabled=False)
 
     def load_model(self, model: SegmenterModel):
-        self.combo_channels.addItems(model.channel_list)
-        self.combo_workflows.addItems(model.workflows)
+        """
+        Load and display data from model
+        Inputs:
+            model: the model to load
+        """
+        self.update_layers(model.layers, model.selected_layer_name)
+        self.update_channels(model)
+        self._load_workflows(model)
+
+    def update_layers(self, layers: List[str], selected_layer_name: str = None):
+        """
+        Update / repopulate the list of selectable layers
+        Inputs:
+            layers: List of layer names
+            selected_layer_name: (optional) name of the layer to pre-select
+        """
+        if layers is None or len(layers) == 0:
+            self.load_image_warning.setVisible(True)
+            self.combo_layers.setEnabled(False)
+        else:
+            self._reset_combo_box(self.combo_layers)
+            self.combo_layers.addItems(layers)
+            if selected_layer_name is not None:
+                self.combo_layers.setCurrentText(selected_layer_name)
+            self.combo_layers.setEnabled(True)
+            self.load_image_warning.setVisible(False)
+
+    def update_channels(self, channels: List[str]):
+        """
+        Update / repopulate the list of selectable channels
+        Inputs:
+            channels: List of channel names
+        """
+        # TODO load channels and update UI state -> https://github.com/AllenCell/napari-aicssegmentation/issues/24
+        pass
+
+    def _load_workflows(self, workflows):  # workflows: List[aicssegmentation.WorkflowStep]
+        # TODO generate workflow grid from list of workflows
+        # -> https://github.com/AllenCell/napari-aicssegmentation/issues/26
+        pass
+
+    def _reset_combo_box(self, combo: QComboBox):
+        """
+        Reset a combo box to its original state, keeping the header but removing all other items
+        """
+        if combo.count() > 0:
+            header = combo.itemText(0)
+            combo.clear()
+            combo.addItem(header)
+
+    def _dropdown_row(self, number: int, placeholder: str, enabled=False) -> FormRow:
+        """
+        Given the contents of a dropdown and a number for the label, return a label and a QComboBox
+        widget that can be used to create a row in a QFormLayout
+        """
+        label = f"{number}."
+
+        dropdown = QComboBox()
```

Comment:
Styling nitpick for something that got lost during transfer: We need to give the dropdown a minimum width with `dropdown.setMinimumWidth()`. I used `360`